datafusion/sqllogictest/test_files/union_by_name.slt

```diff
@@ -152,22 +152,22 @@ NULL 4
 # Limit
 
 query III
-SELECT 1 UNION BY NAME SELECT * FROM unnest(range(2, 100)) UNION BY NAME SELECT 999 ORDER BY 3, 1 LIMIT 5;
+SELECT 1 UNION BY NAME SELECT * FROM unnest(range(2, 100)) UNION BY NAME SELECT 999 ORDER BY 3, 1, 2 LIMIT 5;
```

Comment:
Previously the unnest column was incorrectly #3. Now it is correctly #2 so the result here has changed.

The extra ordering is to order the unnest values for a deterministic result.